Dockerfile

```diff
@@ -208,7 +205,8 @@ RUN . $NVM_DIR/nvm.sh \
     && nvm install "$NODEJS_VERSION" \
     && nvm alias default "$NODEJS_VERSION" \
     && nvm use default \
-    && npm install --global npm@$NPM_VERSION bower@$BOWER_VERSION pnpm@$PNPM_VERSION yarn@$YARN_VERSION
+    && npm install --global bower@$BOWER_VERSION corepack@latest \
```

Comment:
Can we omit "@latest" as just "corepack" should also install the latest version, or?